main.nf

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env nextflow
+
+if(params.help) {
+    usage = file("$baseDir/USAGE")
+
+    cpu_count = Runtime.runtime.availableProcessors()
+    bindings = ["atlas_config":"$params.atlas_config",
+                "atlas_directory":"$params.atlas_directory",
+                "atlas_centroids":"$params.atlas_centroids",
+                "multi_parameters":"$params.multi_parameters",
+                "minimal_vote_ratio":"$params.minimal_vote_ratio",
+                "wb_clustering_thr":"$params.wb_clustering_thr",
+                "seeds":"$params.seeds",
+                "outlier_alpha":"$params.outlier_alpha"
+                ]
+    engine = new groovy.text.SimpleTemplateEngine()
+    template = engine.createTemplate(usage.text).make(bindings)
+    print template.toString()
+    return
+}
+
+log.info "SCIL RecobundlesX pipeline"
+log.info "=========================="
+log.info ""
+log.info "Start time: $workflow.start"
+log.info ""
+
+log.debug "[Command-line]"
+log.debug "$workflow.commandLine"
+log.debug ""
+
+log.info "[Git Info]"
+log.info "$workflow.repository - $workflow.revision [$workflow.commitId]"
+log.info ""
+
+log.info "Options"
+log.info "======="
+log.info ""
+log.info "[Atlas]"
+log.info "Atlas Config: $params.atlas_config"
+log.info "Atlas Anat: $params.atlas_anat"
+log.info "Atlas Directory: $params.atlas_directory"
+log.info "Atlas Centroids: $params.atlas_centroids"
+log.info ""
+log.info "[Recobundles options]"
+log.info "Multi-Parameters Executions: $params.multi_parameters"
+log.info "Minimal Vote Percentage: $params.minimal_vote_ratio"
+log.info "Whole Brain Clustering Threshold: $params.wb_clustering_thr"
+log.info "Random Seeds: $params.seeds"
+log.info "Outlier Removal Alpha: $params.outlier_alpha"
+log.info ""
+log.info ""
+
+log.info "Input: $params.root"
+root = file(params.root)
+/* Watch out, files are ordered alphabetically in channel */
+    in_data = Channel
+        .fromFilePairs("$root/**/{*tracking.trk,*anat.nii.gz}",
+                        size: 2,
+                        maxDepth:2,
+                        flat: true) {it.parent.name}
+
+    atlas_centroids = Channel.fromPath("$params.atlas_centroids/*.trk")
```

Comment:
maybe: `if atlas_centroids`

Reply:
I have to define atlas_centroids to an empty channel anyway.
So if would be 
```
if $params.atlas_centroids
   atlas_centroids = Channel.fromPath(...)
else
   atlas_centroids = Channel.empty()
```
Is that what you both want ?